.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="11" project-jdk-type="JavaSDK">
```

Comment:
Are you using your systems JDK rather than the one bundled with AndroidStudio? Every time one does a gradle sync it should revert back to 1.8. I can leave out this file but it may be worth checking your setup.